doc/api/vm.md

```diff
@@ -135,11 +144,13 @@ function add(a, b) {
 const x = add(1, 2);
 `);
 
-const cacheWithoutX = script.createCachedData();
+const cache1 = script.createCachedData();
```

Comment:
Perhaps the name can updated to `cacheWithoutAdd`? Also it might not be quite useful to mention `stub` here without explaining what it is, so maybe ``In `cacheWithoutAdd` the function `add()` is marked for full compilation upon invocation`` would suffice.

Reply:
Thanks for the suggestion! Sounds reasonable to me. Updated.